### DISCUSSION TITLE
How to remove 🔗 LinkStack from title?

### DISCUSSION THREAD

**Discussion by GustelDE** [JulianPrieber/littlelink-custom#731] [Feature requests]
Hi, Id like to know how I can remove the "🔗 LinkStack" from title.
I found in the advanced-config.php (https://github.com/LinkStackOrg/LinkStack/blob/main/storage/templates/advanced-config.php) this section:

    // Overwrites default page title after the LinkStack name on LinkStack pages.
    // Example: "admin 🔗 LinkStack"
    //                 ⤌----------⤍
    //                 ⬑ What you can change with this setting.
    'linkstack_title' => '',

But if I enter here something or not, I changes nothing.
I use the docker version. I removed the container, created new, restarted... nothing.

**Comment by JulianPrieber** [comment]
@lastsamurai26

**Comment by JulianPrieber** [comment]
We may have to look into this. I'm not sure if it still works

**Reply by lastsamurai26** [reply]
if you change 
'linkstack_title' => '', 
to 
'linkstack_title' => ' ',

then the  🔗 LinkStack is gone (:

**Comment by GustelDE** [comment]
Hi,

'linkstack_title' => ' ', does not work.
Tried this multiple times, with the .env CUSTOM_META_TAGS=false or CUSTOM_META_TAGS=true
and: '', ' ', 'xxx'

NOTHING!!
Please provide step-by-step guide how I can get rid of this.
I use the docker-version. Multiple restarts and even re-creates show zero, null effect.

**Reply by lastsamurai26** [reply]
Login as Admin 
go to Admin -> Config 
-> Tab on "Advanced Config"
search for   'linkstack_title' => ' ',
and change '' to ' ' <- between ' and ' is a Space 

Before
<img width="144" alt="Bildschirmfoto 2024-08-01 um 17 10 37" src="https://github.com/user-attachments/assets/3bd14875-3684-4966-bf65-b041ebe075ce">
After
<img width="119" alt="Bildschirmfoto 2024-08-01 um 17 10 52" src="https://github.com/user-attachments/assets/75826c4f-8d32-40bd-aef4-2a412a58f60c">

**Reply by GustelDE** [reply]
Thank you!!

I was editing the file under this path: storage/templates/advanced-config.php
Not in the Web-GUI.
